src/simplewallet/simplewallet.cpp

```diff
@@ -8456,8 +8456,8 @@ bool simple_wallet::get_transfers(std::vector<std::string>& local_args, std::vec
         }
         else
         {
-          uint64_t current_time = static_cast<uint64_t>(time(NULL));
-          uint64_t threshold = current_time + (m_wallet->use_fork_rules(2, 0) ? CRYPTONOTE_LOCKED_TX_ALLOWED_DELTA_SECONDS_V2 : CRYPTONOTE_LOCKED_TX_ALLOWED_DELTA_SECONDS_V1);
+          const uint64_t adjusted_time = m_wallet->get_daemon_adjusted_time();
+          uint64_t threshold = adjusted_time + (m_wallet->use_fork_rules(2, 0) ? CRYPTONOTE_LOCKED_TX_ALLOWED_DELTA_SECONDS_V2 : CRYPTONOTE_LOCKED_TX_ALLOWED_DELTA_SECONDS_V1);
```

Comment:
Looks like terminology confusion again, since 'adjusted_time' has been changed to mean 'median time of last 60 blocks'. Which makes the behavior here change.

Reply:
Looks like it was already broken. I'm not sure why it's adding that define. The threshold really should be adjusted_time. If you can't think of why the define gets added either, I'll remove.

Reply:
Not sure what you are referring to. Define?

Reply:
I think we should rename it to `locktime_median_time_past` or something along those lines to avoid confusion.

Reply:
By define, I meant adding CRYPTONOTE_LOCKED_TX_ALLOWED_DELTA_SECONDS_V2 etc. I don't see what purpose this has.

Reply:
Is this a Chesterton's fence? It does feel like an idea that didn't get fully fleshed out.

Reply:
On second thought, it doesn't hurt anything either. So as you imply, safer to keep it in at least for now.

Reply:
The name seems clear to me. It's getting the adjusted time from the daemon, not the current time.